content/tests/components/recipes/details/RecipeDetails.test.js

```diff
@@ -94,6 +95,23 @@ describe("The `RecipeDetails` component", () => {
     return /** @type import("devtools/types/recipes").RecipeV3<import("devtools/types/arguments").BranchedAddonStudyArguments> */ (recipe);
   };
 
+  const extensionSetup = () => {
+    const extensions = extensionFactory.buildCount(2);
+    const mockFetchExtension = jest
+      .fn()
+      .mockImplementation((i) => extensions[i % 3]);
```

Comment:
`i % 3` here probably isn't right, since there are only two extensions. This should probably be `.mockImplementation((i) => extensions[i % extensions.length]);`

Reply:
ooh right right